testonly/integration/clientserver.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"database/sql"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"strings"
+
+	"github.com/google/trillian"
+	"github.com/google/trillian/extension/builtin"
+	"github.com/google/trillian/server"
+	"github.com/google/trillian/util"
+	"google.golang.org/grpc"
+)
+
+const (
+	createSQLFile = "../storage/mysql/storage.sql"
+	mysqlURI      = "root@tcp(127.0.0.1:3306)/"
+)
+
+// Env is a test environment that contains both a server and a connection to it.
```

Comment:
```
type Env struct {
  grpcServer *grpc.Server
  logServer  *server.TrillianLogRPCServer
  ClientConn *grpc.ClientConn
 }
```
Doesn't the `logServer *server.TrillianLogRPCServer` mean that this test environment could not be used to test a Trillian map server?  Or am I missing something?